client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx

```diff
@@ -169,12 +192,20 @@ class FrontsCAPISearchInput extends React.Component<
 
     const {
       q,
-      selected: { tags, sections }
+      selected: { tags, sections, desks, ratings }
     } = this.state;
 
-    const searchTermsExist = !!tags.length || !!sections.length || !!q;
+    const searchTermsExist =
+      !!tags.length || !!sections.length || !!desks.length || !!q;
+
+    const allTags = tags.concat(sections, desks, ratings);
```

Comment:
should the ratings be concatenated with the tags or should they be stored on the state separately?

Reply:
This is need for the Commission Desks dropdown query